lumicks/pylake/population/dwelltime.py

```diff
@@ -98,7 +125,18 @@ def calculate_stats(self, key, component, alpha=0.05):
         upper = np.quantile(data, 1 - (alpha / 2))
         return mean, (lower, upper)
 
+    @deprecated(
+        reason=(
+            "This method has been renamed to more closely match its behavior. Use "
+            ":meth:`DwelltimeBootstrap.hist()` instead."
+        ),
+        action="always",
+        version="0.13.3",
+    )
     def plot(self, alpha=0.05, n_bins=25, hist_kwargs=None, span_kwargs=None, line_kwargs=None):
+        self.hist(alpha, n_bins, hist_kwargs, span_kwargs, line_kwargs)
+
+    def hist(self, alpha=0.05, n_bins=25, hist_kwargs=None, span_kwargs=None, line_kwargs=None):
```

Comment:
Ah, sorry for not noticing this in my earlier review, but now would be a really good time to make these keyword arguments if it's preferable (with the name change). I imagine that especially the `kwargs` ones might benefit from this for if we ever get more arguments in between?